site/packages/aa-accounts/light-account/actions/transferOwnership.md

```diff
@@ -50,7 +50,7 @@ const transferredClient = await createLightAccountClient({
 
 ### `Promise<0x${string}>`
 
-A Promise containing the hash of either the UO or transaction containing the UO which transferred ownership of the smart account's owner.
+A `Promise` containing the hash of either the UO or transaction containing the UO which transferred ownership of the smart account's owner.
```

Comment:
(Doesn't need to be changed in this PR) Wait, how does this API work? When does it return the UO hash and when the transaction hash? Those are two very different things and feels like it needs clarity.